### DISCUSSION TITLE
CORE_ASPECT_RATIO definition

### DISCUSSION THREAD

**Discussion by oharboe** [The-OpenROAD-Project/OpenROAD-flow-scripts#730] [General] [answered]
When I run a floorplan, I see that CORE_ASPECT_RATIO is height / width, which is surprising, considering if you google aspect ratio, you get width : height, e.g. a 16:9 TV.

I take it this is the way it is supposed to be, but I was wondering if there's anything more to be said on the subject than that OpenROAD uses height / width?

**Comment by maliberty** [comment, marked as answer]
This just strikes me as a bug since it is contrary to standard usage.  Fixing it will be an incompatible change but I don't think it is heavily used.

@vvbandeira @vijayank88 are there many test cases using CORE_ASPECT_RATIO?

**Comment by vijayank88** [comment]
It's explained here: https://openroad-flow-scripts.readthedocs.io/en/latest/user/FlowVariables.html#id2
Test case [config.mk](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/2f572bcc84a401f64ce9ff8b1a66337253e4936e/flow/designs/sky130hs/aes/config.mk)
```
core_area = design_area / (utilization / 100)
core_width = sqrt(core_area / aspect_ratio)
core_height = core_width * aspect_ratio
core = ( core_space_left, core_space_bottom )
      ( core_space_left + core_width, core_space_bottom + core_height )
die =  ( 0, 0 )
      ( core_width + core_space_left + core_space_right,
        core_height + core_space_bottom + core_space_top )
```

**Reply by oharboe** [reply]
I see. Thanks!
